src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -166,10 +166,10 @@
       [:div {}
        [comps/Button {:text "Sign In"
                       :href (str "/service/login?path="
-                                 (js/encodeURIComponent (let [hash (nav/get-hash-value)]
+                                 (js/encodeURIComponent (js/decodeURIComponent (let [hash (nav/get-hash-value)]
```

Comment:
I think this value is coming back from orchestration double-encoded. If that is true, this is masking over the problem, which I'd really like us to not do. I'm working on verifying this theory.